### ISSUE TITLE
Intro to Deep Learning

### ISSUE THREAD

**Issue by soltaniehha** [GDS-Education-Community-of-Practice/DSECOP#2]
The focus will be on physics-related datasets. This could be astronomy or other areas. 

Intro to neural networks:
1.        The history of Deep Learning and Neural Networks
2.        The building blocks of Deep Learning and Neural Networks (e.g., perceptrons)
3.        The architecture of Neural Networks
4.        The concept of the cost function and its different forms
5.        Mathematical optimization and differentiation methods for Deep Learning (e.g., automatic differentiation and backpropagation methods)
Model tuning best practices:
1.        The concept of overfitting and regularization in Deep Learning
2.        The concept of Hyper-parameters
Advanced deep learning topics
1.        Challenges in training Deep Learning models (e.g., vanishing gradient problem)
2.        Types of Deep Learning models (e.g., convolutional neural networks, …)"
